### DISCUSSION TITLE
Remix + PocketBase

### DISCUSSION THREAD

**Discussion by ausminternet** [pocketbase/pocketbase#3957] [Show and tell]
I wanted to have a closer look at Remix and PocketBase a few days ago and therefore I created an example app with complete server side registration and authentication flow (including OAuth), using Remix and PocketBase:

https://github.com/ausminternet/remix-pocketbase-starter

Additionally there are settings pages to update your profile, change your email or your password. Including verification flows via email from PocketBase.

I'm happy to get your feedback, since it's my first ever created Remix and PocketBase app 🙂

**Comment by ausminternet** [comment]
You can now see it live in action: https://remixpb.uber.space/

**Reply by deadcoder0904** [reply]
thanks for this. was looking for a remix example & found it. lets see if i go with pocketbase :)
